cypress/integration/jsTaskParseListDOM.spec.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+describe('List', () => {
+  beforeEach(() => {
+    cy.visit('');
+  });
+
+  it('should render list', () => {
+    cy.get('ul');
+  });
+
+  it('should have 11 items', () => {
+    cy.get('ul > li').should('have.length', 11);
+  });
+
+  it('should be sorted by salary DSC', () => {
+
+    cy.get('ul > li').then(($ul) => {
+      const list = [...$ul].map((row) => row.dataset.salary
+        .replace('$', '').replace(',', ''));
```

Comment:
break line at higher level concepts.
```suggestion
      const list = [...$ul].map(
          (row) => row.dataset.salaryreplace('$', '').replace(',', ''));
```